documentation/github-actions-docs-example.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'GitHub Action Example: Syncing Markdown'
+category: 5f7ce9e3a5504d0414d024c2
+---
+
+Do you have Markdown files stored on GitHub? With [the `rdme` GitHub Action](https://docs.readme.com/docs/rdme), you can sync them to ReadMe every time they're updated in GitHub. Let's go over how to set this up!
+
+## "Automagical" Workflow File Generation 🪄
+
+To set up a GitHub Actions workflow for syncing a directory of Markdown docs, the fastest and easiest way to do so is by running the following command on your local machine:
+
+```sh
+rdme docs [path-to-directory-of-markdown] --github
+```
+
+This will scan the directory for Markdown files, sync them to ReadMe, and then create your GitHub Actions workflow file. Once that's done, create your [repository secret](url), push your workflow file to GitHub, and get syncing! 🎊
+
+## Constructing a GitHub Actions Workflow File By Hand 🔨
+
+> 🚧 Wait — there’s an easier way to do this!
+>
+> The information below is useful if you have experience using GitHub Actions, but for most folks, we recommend using the steps [detailed above](#automagical-workflow-file-generation-🪄). The `rdme` CLI will ask you a few questions before automatically creating the GitHub Actions workflow file for you, complete with every parameter you need to get syncing.
+
+In order to construct the file by hand, you'll first want to grab a project version to ensure that your docs sync to the right place in your developer hub. That version will be passed via the `--version` flag. See below for a full example:
+
+```yml
+name: Sync `documentation` directory to ReadMe
+
+# Run workflow for every push to the `main` branch
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout this repo
+        uses: actions/checkout@v3
+
+      # Run GitHub Action to sync docs in `documentation` directory
+      - name: GitHub Action
+        # We recommend specifying a fixed version, i.e. @RDME_VERSION
```

Comment:
Using `@RDME_VERSION` in this comment seems a little hard to understand. I know why you don't want to hardcode 8.0.0 in here, but I feel like people are going to gloss over this thinking that it's ok to leave it at `RDME_VERSION`. What about a fake version that doesn't exist, like 7.6.0? Then if they get an error from GitHub like "Can't find that version" at least it's a little easier to figure out what's wrong.

(Same comment applies to documentation/github-actions-openapi-example.md.)

Reply:
Yeah unfortunately we don't have an API for updating variable defaults but it's something we've chatted about and probably should look into (especially with the recent Personalized Docs launch)